### PR TITLE
[KOGITO-4503] - Update to Quarkus 1.12 on artifacts broke the images

### PR DESCRIPTION
**Pull request by spolti**
Many thanks for submitting your Pull Request :heart:! 
See https://issues.redhat.com/browse/KOGITO-4503

Depends on https://github.com/kiegroup/kogito-examples/pull/594

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by spolti** [comment]
depends on: https://github.com/kiegroup/kogito-images/pull/378

**Comment by evacchi** [comment]
guys as mentioned in a few places, we are going to downgrade to 1.11.x so I think we should still allow to support the alternative file paths, but we should also support the previous layout and/or setup the examples consistently

EDIT: I think the images as rewritten here may work even in that case right? @spolti

**Comment by spolti** [comment]
@evacchi yes.
What do you mean downgrading to 1.11? sry I probably missed this discussion.
the changes on this pr are just to apply the Quarkus 1.12 changes on the dockerfiles. If the plan still be using 1.11.x it is fine as well on the images side.

**Comment by evacchi** [comment]
@spolti yes I'll temporarily downgrade kogito-runtimes to Quarkus 1.11.4; but we will still be contributing to platform (which will advance their version as usual, overriding ours, as long as the cross-builds succeed); so the fix is good, but it should be forward-compatibile (or backwards compatible, depending on how you look at it)

File: dmn-drools-quarkus-metrics/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,10 @@
-FROM quay.io/kiegroup/kogito-quarkus-jvm-ubi8:latest
+FROM quay.io/kiegroup/kogito-runtime-jvm:latest
 
-COPY target/*-runner.jar $KOGITO_HOME/bin
-COPY target/lib $KOGITO_HOME/bin/lib
+COPY target/quarkus-app/lib/ $KOGITO_HOME/bin/lib/
```
Comment (radtriste) [inline dmn-drools-quarkus-metrics/Dockerfile:5]: This should not be needed and will be addressed/removed by https://issues.redhat.com/browse/KOGITO-4609

**Comment by radtriste** [comment]
@evacchi @spolti isn't this duplicate from https://github.com/kiegroup/kogito-examples/pull/601 ?

**Comment by radtriste** [comment]
@spolti Why did you close that one ? I think we still need it until https://issues.redhat.com/browse/KOGITO-4609 is done

**Review by radtriste — APPROVED** [review]

**Comment by evacchi** [comment]
fyi master is now Quarkus 1.11

**Comment by radtriste** [comment]
@spolti we can merge that one or ?

**Comment by radtriste** [comment]
Jenkins retest this

**Review by evacchi — APPROVED** [review]

**Comment by spolti** [comment]
@radtriste i closed because I thought it wouldn't be needed anymore as Quarkus was downgraded.
It still needed as well due fast-jar config. Thanks for re-opening and merging this.

**Pull request merged**